pages/conduct.html

```diff
@@ -37,9 +38,9 @@
 <ol>
   <li>
     <p>
-      Harassment includes offensive verbal comments related to gender,
+      Harassment includes offensive verbal or written comments related to gender,
       sexual orientation, disability, physical appearance, body size,
-      race, religion, sexual images in public spaces, deliberate
+      race, or religion, sexual images in public spaces, deliberate
```

Comment:
@wking Thanks for the comment; I agree. I was wary about doing anything more than the bare minimum to the CoC, because it can be such a sensitive document and because I've had strong opinions in the discussion that led to me looking at it. I think I'll leave this PR as "fixing the fact that it simply didn't cover online stuff" (and, without this "or", that it didn't quite make sense) and, if I get a chance in the next few days, file a second PR that improves its readability. That way the owners can accept this but not that, if they choose.